core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -631,6 +631,17 @@ secret_configs:
       - cluster: api.ci
         namespace: ci
         name: operator-manifests-test-credentials
+  # pull secret on build farms
+  - from:
+      ".dockerconfigjson":
+        bw_item: quay.io
+        attachment: pull-secret
+    to:
+      - cluster_groups:
+        - build_farm
```

Comment:
This is the cluster-wide secret, right? Can we just set that on app.ci which is OSD managed?